sql/derived_tables/po_lines_locations.sql

```diff
@@ -1,40 +1,40 @@
-DROP TABLE IF EXISTS po_lines_locations;
+/* This query creates a local table for purchase order line locations including location quantity and type.
```

Comment:
I wonder if the description should say "derived table" instead of "local table." In my mind, those are two different things.

Reply:
Good catch! Have changed that.